action.yml

```diff
@@ -8,6 +8,7 @@ inputs:
   github-token:
     description: Github token
     required: true
+    default: ${{ github.token }}
```

Comment:
Can you point me to some documentation on `github.token`. I haven't been able to find anything.